### ISSUE TITLE
Errant [ character makes a required argument look optional

### ISSUE THREAD

**Issue by natebosch** [dart-lang/dartdoc#2082]
https://api.dart.dev/stable/2.6.1/dart-core/Uri/Uri.https.html

```
Uri.https(
    String authority, [
    String unencodedPath, [
    Map<String, String> queryParameters
])
```

The `[` before `String unencodedPath` made me think it was part of the optional positional arguments, but it is a required argument.

**Comment by srawlins** [comment]
Also reported for [List.fillRange](https://api.dartlang.org/stable/2.6.1/dart-core/List/fillRange.html) at https://github.com/dart-lang/sdk/issues/39587

**Comment by jcollins-g** [comment]
The bracket chaos is already fixed in #2075.  Will try and publish shortly since everyone seems suddenly to be noticing that this was broken now.
